File: matlab/black_and_white.m
function img = image_noire(h, l)
    img = zeros(h, l);
end

function img = image_blanche(h, l)
    img = ones(h, l);
end

function img = creerImgBlancNoir(h, l)
    img = zeros(h, l);
    for i = 1:h
        for j = 1:l
            img(i, j) = mod(i + j, 2);
        end
    end
end

function img_neg = negatif(img)
    img_neg = 1 - img;
end

File: python/black_and_white.py
import matplotlib.pyplot as plt
import numpy as np

def image_noire(h, l):
	return np.zeros((h, l), dtype=int)

def image_blanche(h, l):
	return np.ones((h, l), dtype=int)

def creerImgBlancNoir(h, l):
	img = np.zeros((h, l), dtype=int)
	for i in range(h):
		for j in range(l):
			img[i][j] = (i + j) % 2
	return img

def negatif(img):
	return 1 - img